app/controllers/application_controller.rb

```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  self.abstract_class = true
 end
```

Comment:
- [ ] [REQUIRED] Kindly let's ensure that the ApplicationController contains `current_user` method.